src/dashboard/pages/login/login_page.py

```diff
@@ -0,0 +1,168 @@
+"""The layout component of the login page."""
+
+import dash
+from dash import callback, dcc, html
+from dash.dependencies import Component, Input, Output, State
+
+PORT = 8000
+ADDRESS = "127.0.0.1"
+PATH = "/login"
+
+dash.register_page(__name__, path=PATH, nav_item=False)
+
+# sets the colors of the login page
+colors = {
```

Comment:
See issue #80.

Reply:
I could try to fix the issue after this PR gets accepted